docs/evaluation/simulate.rst

```diff
@@ -62,11 +94,12 @@ time. So, we first create a :py:class:`~sherpa.astro.data.DataPHA` object with
 the correct exposure time, but we can leave the settings for channel
 and counts empty, because these will be filled in by the simulation::
 
-  >>> from sherpa.astro.data import DataPHA
-  >>> from sherpa.astro.io import read_arf, read_rmf, read_pha
-  >>> data = DataPHA(name='any', channel=None, counts=None, exposure=10000.)
-  >>> data.set_arf(read_arf(data_dir + '9774.arf'))
-  >>> data.set_rmf(read_rmf(data_dir + '9774.rmf'))
+   >>> from sherpa.astro.data import DataPHA
+   >>> from sherpa.astro.io import read_arf, read_rmf, read_pha
+   >>> data = DataPHA(name='simulation', channel=None, counts=None, exposure=10000.)
+   >>> data.set_arf(read_arf(data_dir + '9774.arf'))
+   >>> data.set_rmf(read_rmf(data_dir + '9774.rmf'))
+   >>> data.channel = np.arange(1, 1025, dtype=np.int16)
```

Comment:
I wonder if it's more intuitive to have a channel variable first, instead of setting it to None when the Data object is created.
```suggestion
   >>> channels = np.arange(1, 1025, dtype=np.int16)
   >>> data = DataPHA(name='simulation', channel=channels, counts=None, exposure=10000.)
   >>> data.set_arf(read_arf(data_dir + '9774.arf'))
   >>> data.set_rmf(read_rmf(data_dir + '9774.rmf'))
```

Reply:
Allowing the channel field to be None has been a thorn in my side, so I'm happy to move documentation away from this approach!